commands/help.js

```diff
@@ -0,0 +1,70 @@
+const Discord = require('discord.js');
+const func= {
+    botMessage: function(message,args) {
+        const data = [];
+		//const { commands } = this.client;
+        console.log(this.client);
+        
+        const commands =[{
+            name: "github",
+            description:"To get you started with posting comments on githubs issues/PR."
+        },
+        {
+            name: "github-info",
+            description:"Lets you enter your github personal token to authenticate the bot."
+        },
+        {
+            name: "github-comment-issue",
+            description:"Tells the bot to post a comment on an issue."
+        },
+        {
+            name: "github-issue-number",
+            description:"Tells the bot which issue you want to comment you."
+        },
+        {
+            name: "github-post-comment",
+            description:"Tells the bot what you want to comment on the issue."
+        }];
+        if (!args.length) {
+            data.push('Here\'s a list of all my commands:');
+            data.push(commands.map(command => command.name).join(', '));
+            data.push(`\nYou can send \`-help [command name]\` to get info on a specific command!`);
```

Comment:
Likewise, using single `'` or double `"` quotes here would avoid you having to escape the backticks `` ` ``